pages/volunteers.html

```diff
@@ -17,7 +19,6 @@ <h3 class="card-header"><a id="volunteers">VOLUNTEERS</a></h3>
             >
             </div>
             <h4>Become a volunteer</h4>
```

Comment:
This should be moved, removed, or made into an email link to the volunteering email?